addons/controllers/testdata/test-vsphere-cpi-non-paravirtual.yaml

```diff
@@ -61,6 +61,8 @@ data:
 apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
 kind: VSphereCluster
 metadata:
+  labels:
+    cluster.x-k8s.io/cluster-name: test-cluster-cpi
   name: test-cluster-cpi
```

Comment:
Could we update the cluster to mimic how capv adds a suffix?

Reply:
Thanks for pointing that out. `VSphereCluster` names are updated.

Reply:
Why don't we get these information from Cluster variables? This is an example of variable vcenter 
```
- name: vcenter
      value:
        cloneMode: fullClone
        datacenter: /dc0
        datastore: /dc0/datastore/sharedVmfs-0
        folder: /dc0/vm
        network: /dc0/network/VM Network
        resourcePool: /dc0/host/cluster0/Resources
        server: 10.187.96.70
        storagePolicyID: ""
        template: /dc0/vm/photon-3-kube-v1.23.8+vmware.2-tkg.2-81d1a7892ad39f017fbaf59f9907cbe7
        tlsThumbprint: 1E:BB:09:45:FD:02:10:17:FD:55:8D:D1:05:8B:3F:A8:91:DD:29:0E
```